src/lambda-edge/refresh-auth/index.ts

```diff
@@ -36,8 +36,9 @@ export const handler: CloudFrontRequestHandler = async (event) => {
       cookies.refreshToken
     );
 
-    const headers: { "Content-Type": string; Authorization?: string } = {
+    const headers: { "Content-Type": string; Authorization?: string; "User-Agent":string } = {
```

Comment:
Let's just change this type to `OutgoingHttpHeaders` -- bit annoying to spell all keys we use out like this.